src/css/custom.css

```diff
@@ -284,12 +302,19 @@ article.opensource > a > p::before {
   content: '🛠️';
 }
 
-a.menu__link, a.card {
+a.menu__link,
+a.card,
+li.menu__list-item,
+li.menu__list-item > div {
   position: relative;
 }
-.enterprise a:hover::after,
-.cloud a:hover::after,
-.opensource a:hover::after {
+li.enterprise > div:hover::after,
```

Comment:
Had to put this `::after` onto a different element, because you can’t have two `::after`s on the same element